test/hotspot/jtreg/runtime/cds/appcds/jcmd/JCmdTestFileSafety.java

```diff
@@ -41,8 +41,25 @@
 import jdk.test.lib.cds.CDSTestUtils;
 import jdk.test.lib.apps.LingeredApp;
 import jdk.test.lib.Platform;
+import jdk.test.lib.process.OutputAnalyzer;
 
 public class JCmdTestFileSafety extends JCmdTestDumpBase {
+    static final String promtStdout = "please check stdout file";
+    static final String promtStderr = "or stderr file";
```

Comment:
Could you change the variable names to `promptStdout` and `promptStderr` instead?

Please also add the bug id to the `@bug`.